internal/config/config_test.go

```diff
@@ -46,7 +46,8 @@ func TestBadConfigs(t *testing.T) {
 		},
 		{
 			"unknown fields",
-			"json: unknown field \"foo\"",
+			`yaml: unmarshal errors:
```

Comment:
I'm not happy about this error changing. Going forward, we shouldn't be exposing an error message directory from a dependency.